filesystem.h

```diff
@@ -19,12 +19,19 @@ class Filesystem
     void mkdir(stringstream& ss);
     void touch(stringstream& ss);
 
+    void ls(stringstream& ss);
+
+    std::vector<Directory *> parseRelativePath(string arg);
+    Directory* getRelativeDir(string path);
+
     void cd(stringstream& ss);
     void cdRoot();
     void cdParent();
+    void cdRelativePath(string arg);
     bool cdToDirectory(Directory* newLocation);
```

Comment:
typic peldaja az ilyen elburjanzott takolt dolgoknak. Van 5 db cd fuggveny, ebbol 4 voidot ad vissza, d evan egy bool, a kovetelt argumentumokban is eleg valtozatosak.  Nem latszik, hogy melyik mit kezel pontosan le, Lehet jo volna kicsit atgondolni, refaktoralni.  Pl csak erre ranezve, az ember mondhatna, hogy a `cdRoot` az csak egy `cdRelativePath("/")`, hasonlokeppen a parentre.